### PR TITLE
Added Feature for Registering Issuer

### PR DESCRIPTION
**Pull request by saivittalb**
To test this feature, run the project locally using ```ng serve``` and then change the address to ```localhost:4200/signup``` and also please connect the ```Certificate Front Server URL``` to the ```baseURL in environment.ts``` file.

Screenshot to show the project is running after implementing this feature👇

screen shot 2018-11-21 at 1 24 52 am

### CODE REVIEW

File: src/app/core/signup/signup.component.html [hunk 1]
```diff
@@ -1,3 +1,49 @@
-<p>
-  signup works!
-</p>
+<link href="https://fonts.googleapis.com/css?family=Varela+Round" rel="stylesheet">
```
Comment (nikitadhiman) [inline src/app/core/signup/signup.component.html]: no link href or script src in html files, they should contain only html

File: src/app/core/signup/signup.component.ts [hunk 2]
```diff
@@ -1,15 +1,39 @@
...
-
-  constructor() { }
-
-  ngOnInit() {
+  signupForm: FormGroup;
+  error: string;
+  user: User;
+  expiration_date: Date;
+  constructor(private mAuth: AuthService, private formBuilder: FormBuilder, private router: Router) {
+    this.signupForm = this.formBuilder.group({ username: ['', Validators.required], password: ['', Validators.required] });
```
Comment (nikitadhiman) [inline src/app/core/signup/signup.component.ts]: don't add this in constructor, only dependency injections should be a part of constructor

File: src/app/core/signup_assets/footer.component.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<footer class="footer">
+    <span class="text-center py-3">Copyright ©
+      <a style = "text-decoration: none; color: white;"href="https://codein.withgoogle.com/organizations/jboss-community/"> JBoss Outreach 2018</a>
+    </span>
```
Comment (nikitadhiman) [inline src/app/core/signup_assets/footer.component.html]: please remove this... no href here

File: src/app/core/backend_assets/emailverify.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<?php
+$nameErr = $emailErr = "";
```
Comment (nikitadhiman) [inline src/app/core/backend_assets/emailverify.html]: php?!

Comment (saivittalb) [reply src/app/core/backend_assets/emailverify.html]: Hmm...yeah! How to deal with it?😕

**Pull request closed without merging**